tests/utils/test_search_utils.py

```diff
@@ -84,6 +84,14 @@ def test_anded_expression_2():
                                  'key': 'model',
                                  'type': 'parameter',
                                  'value': "LR"}]),
+    ("tags.version = 'commit-hash'", [{'comparator': '=',
```

Comment:
maybe also write some "failure" condition tests for tags like numeric value, wrong type of comparators, ... etc.